students/hwijeong/westagram/users/views.py

```diff
@@ -25,10 +26,12 @@ def post(self, request):
             is_email_valid(email)
             is_password_valid(password)
             
+            hashed_password = bcrypt.hashpwd(password.encode('utf-8'), bcrypt.gensalt())
+
             user = User.objects.create(
                 name     = name,
                 email    = email,
-                password = password,
+                password = hashed_password.decode('utf-8'),
                 contact  = contact,
                 note     = note
```

Comment:
```suggestion
                note     = note,
```
간신히 찾았습니다 지적할 것이 없습니다